CHANGELOG.md

```diff
@@ -8,6 +8,13 @@
 
 ### Fixed
 
+- **decidim-participatory_processes**: Fix some linterns errors arising from capybara upgrade [\#6197](https://github.com/decidim/decidim/pull/6197)
```

Comment:
Can you apply this correction in all the changelog entries please?
```suggestion
- **decidim-participatory_processes**: Fix some linters errors arising from capybara upgrade [\#6197](https://github.com/decidim/decidim/pull/6197)
```

Reply:
Thanks! I've fixed this errors